### ISSUE TITLE
copy and paste is not supported

### ISSUE THREAD

**Issue by Ahmed707707** [stonesam92/ChitChat#95]
copy and paste option is not supported!!

**Comment by joostrijneveld** [comment]
Can you be a bit more specific? I cannot seem to reproduce this.. What are you trying to copy?

**Comment by Ahmed707707** [comment]
copy and paste text is fine but it doest work with image.

**Comment by RexFuzzle** [comment]
I have the same issue. When trying to paste an image it does not work. The only way to do it is to drag it, and this only works sometimes (Haven't been able to figure out when exactly.)

**Comment by GraemeF** [comment]
Would be very nice if it was possible to paste an image into ChitChat. I'm always having to save to a file and then drag it in, which is a bit of a faff.

**Comment by mrgluek** [comment]
Pasting would be very useful (ex. for screenshots).

**Comment by harrykiselev** [comment]
yep, +1 for pasting images from buffer, I often use screenshot sending in my conversations :)
